TODO/tools.md

```diff
@@ -26,44 +26,13 @@ slug: tools
 
 ## Document Processing
 
-- [Squoosh](https://squoosh.app): Image compression web app provided by Googl
-- [PDF2JPG](https://pdf2jpg.net): Free PDF to JPG converter
-- [Smallpdf](https://smallpdf.com): Solve any problems with PDF (conversion, compression, rotate, merge, unlock)
-- [ILovePDF](https://www.ilovepdf.com): Every tool you need to work with PDFs
-- [ILoveIMG](https://www.iloveimg.com): Every tool you need to work with images
-- [Convertio](https://convertio.co): Converts for fonts, video, audio, presentation, image, ebook et al
 - [Merge pdf in linux](https://blog.seisman.info/merge-pdf)
 - [cpdf](https://blog.seisman.info/cpdf): Trans-platform PDF process tookit
 - [PDFtk](https://blog.seisman.info/pdftk): Trans-platform PDF process tookit
```

Comment:
Those two tools are already included in https://seismo-learn.org/links/tools/. Shall we move the blog content to seism-learn, e.g., software?

Reply:
I really don't want to include any non-seismo software in the sites.